src/utils/flattenMetadataQueryResponse.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @flow
+ * @file File for some simple dom utilities
+ * @author Box
+ */
+
+import omit from 'lodash/omit';
+import { getFileExtension } from './file';
+
+type ConvertMetadataParams = {
+    item: Object,
+    metadata: Object,
+};
+
+const convertISODateToSeconds = (dateString: string) => {
+    return dateString ? new Date(dateString).getTime() / 1000 : null;
+};
+
+const convertMetadata = (metadata: ConvertMetadataParams) => {
+    let md = {};
+
+    Object.keys(metadata).forEach(scope => {
+        Object.keys(metadata[scope]).forEach(templateKey => {
+            const nonconformingInstance = metadata[scope][templateKey];
+            const data = omit(nonconformingInstance, [
+                '$id',
+                '$parent',
+                '$type',
+                '$typeScope',
+                '$typeVersion',
+                '$version',
+            ]);
+
+            md = {
```

Comment:
This is going to take the last value in this nested structure. Is that intentional? The sort order of `Object.keys` isn't guaranteed. Can we use a lodash helper to select the last value?

Reply:
Typical metadata_queries response is of structure:
`{ entries: [ { item: {...}, metadata: { enterprise_<eid>: { <templateKey>: {...} } } } ], next_marker: "markermarkermarker" }`

Because both `enterprise_<eid>` and `<templateKey>` are dynamic key values, getting the actual metadata through `Object.keys` is one way of doing it.